CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## Unreleased
+
+#### Breaking Changes
+
+- `map.featuresAt` and `map.featuresIn` are removed. Use `map.queryRenderedFeatures` or `map.querySourceFeatures` instead. To migrate:
+    - replace `featuresAt` and `featuresIn` with `queryRenderedFeatures`
+    - `queryRenderedFeatures` is synchronous. Remove the callback and use the return value. An async version `queryRenderedFeaturesAsync` exists for really slow queries.
+    - rename the `layer` parameters to `layers` and make it an array of strings.
+    - remove the `radius` paramter. `queryRenderedFeatures` automatically uses style property values when doing queries.
```

Comment:
This was the initial intent of #2052. Style properties like line width, translate, etc. all affect whether an object is under a cursor or not, even with the same coordinates. Previously we would use `radius` because we checked against original coordinates, not rendered representation, but it's no longer needed.